src/target/stm32l4.c

```diff
@@ -583,7 +622,7 @@ static bool stm32l4_attach(target_s *const t)
 	/* Free any previously built memory map */
 	target_mem_map_free(t);
 	/* And rebuild the RAM map */
-	if (device->family == STM32L4_FAMILY_L55x)
+	if ((device->family == STM32L4_FAMILY_L55x) || (device->family == STM32L4_FAMILY_U5xx))
```

Comment:
The parens are unnecessary and suppress a GCC warning about accidental assignment, please drop them:
```suggestion
	if (device->family == STM32L4_FAMILY_L55x || device->family == STM32L4_FAMILY_U5xx)
```